themes/default/assets/sass/_api-nodejs.scss

```
@@ -36,15 +36,15 @@ h3.api {
 ul.api {
     @apply list-none p-0 m-0 overflow-hidden;
 
-    columns: 3 220px;
+    columns: 2 330px;
 
     li {
         @apply truncate mt-0;
 
         a {
             @apply btn truncate bg-transparent inline-block p-1;
```

Similarly we should be able to just say "max-width of 100%" and get the same result:

```suggestion
            @apply btn truncate bg-transparent inline-block p-1 max-w-full;
```

---

oh okay, i was trying to align with what was already there